demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1117 @@
+{
```

Comment:
<p>We now have a <code>datasets</code> module that we can use in our notebooks for downloading datasets like cora:</p><pre class="ql-syntax" spellcheck="false">from stellargraph import datasets
from IPython.display import display, HTML

dataset = datasets.Cora()
display(HTML(dataset.description))
dataset.download()
</pre><p>You can have a look at this notebook for reference: <a href="https://github.com/stellargraph/stellargraph/blob/develop/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb" target="_blank">https://github.com/stellargraph/stellargraph/blob/develop/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
My initial assumption is that we:

- Firstly, ingest the whole dataset into Neo4J database. I will upload the demo notebook for this for the next review request, using the ```dataset``` module).

- Secondly, from the database, we import node features into memory.

So I think it slightly makes more sense to use the ```dataset``` module at the first stage!